kamel-core/build.gradle.kts

```diff
@@ -70,17 +75,38 @@ kotlin {
                 implementation(kotlin("test"))
                 implementation(libs.ktor.client.mock)
                 implementation(libs.kotlinx.coroutines.test)
+                implementation(libs.okio.fakefilesystem)
             }
         }
 
-        val jvmMain by getting {
+        val commonJvmMain = create("commonJvmMain") {

```

Comment:
I'm not sure if I like "commonJvmMain".... I might change that. But don't worry about that for now

Reply:
I also don't like this. I don't like how `jvmMain` is a common source set for JVM and desktop but if you also introduce jvmDesktop it is not common anymore.  JetBrains should keep the `jvmMain` as a common source set for Android and JVM regardless if you support a desktop or not.

Reply:
I just asked about it on slack and got this link.... I'm confused about what they're doing in 1.9.20. But hopefully things don't break:

https://kotlinlang.slack.com/archives/C3PQML5NU/p1697102152200739